### PR TITLE
init order service

### PR DESCRIPTION
**Pull request by LEHOANGGLAM**
Please review this PR for me.
- Init OrderSerivce with port: 8085
- Create github action
- Add order service in prometheus for tracking
- Add port 8085 to realm-export.json
- Add order location in nginx
- Add query for create order database in postgres_init
Please tell me if there any missing, thank you.

### CODE REVIEW

File: order/src/main/resources/application.properties [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+# Hibernate ddl auto (none, create, create-drop, validate, update)
+spring.jpa.hibernate.ddl-auto = none
+
+#Enable liquibase
+spring.liquibase.enabled=true
+
+# swagger-ui custom path
+springdoc.swagger-ui.path=/swagger-ui
+springdoc.packagesToScan=com.yas.rating
```
Comment (chanhnguyen6975) [inline order/src/main/resources/application.properties:31]: order

Comment (LEHOANGGLAM) [reply order/src/main/resources/application.properties:31]: Thanks

File: nginx/templates/default.conf.template [hunk 2]
```diff
@@ -18,6 +18,9 @@ server {
   location /rating/ {
 	proxy_pass http://rating;
   }
+  location /order/{
```
Comment (toannguyenk) [inline nginx/templates/default.conf.template:21]: space

Comment (LEHOANGGLAM) [reply nginx/templates/default.conf.template:21]: thanks

**Review by thiennn — APPROVED** [review]

**Pull request merged**